lib/readyset.rb

```diff
@@ -13,8 +13,17 @@ def self.configuration
     @configuration ||= Configuration.new
   end
 
+  def self.current_configuration
+    configuration.inspect
+  end
+
+  class << self
```

Comment:
Thoughts on removing these aliases and just renaming the methods to `config` and `current_config`?

Reply:
From what I saw from other gems, this is a solid convention. Just thought it'd be a nice little quality-of-life change.